### PR TITLE
Random clean ups

### PR DESCRIPTION
**Pull request by AJenbo**

### CODE REVIEW

File: Source/msg.cpp [hunk 1]
```diff
@@ -2087,7 +2087,7 @@ DWORD OnNakrul(const TCmd *pCmd)
 		}
 		IsUberRoomOpened = true;
 		Quests[Q_NAKRUL]._qactive = QUEST_DONE;
-		monster_some_crypt();
+		WeakenNaKrul();
```
Comment (yuripourre) [inline Source/msg.cpp:2090]:
```suggestion
		AwakenNaKrul();
```

File: Source/monster.h [hunk 2]
```diff
@@ -276,7 +276,7 @@ void PrepareUniqueMonst(Monster &monster, int uniqindex, int miniontype, int bos
 void InitLevelMonsters();
 void GetLevelMTypes();
 void InitMonsterGFX(int monst);
-void monster_some_crypt();
+void WeakenNaKrul();
```
Comment (yuripourre) [inline Source/monster.h:279]:
```suggestion
void AwakenNaKrul();
```

Comment (yuripourre) [reply Source/monster.h:279]: I never played Hellfire but I think this suppose to enable NaKrul instead of reducing his power, right?

Comment (AJenbo) [reply Source/monster.h:279]: No. He is locked in a room. If you just open the door he will be a full strength but if you trigger spell books out side his room first he will be weakened making the fight much easier.

**Review by yuripourre — DISMISSED** [review]

**Pull request merged**